.golangci.yml

```diff
@@ -29,3 +29,8 @@ linters:
     - vet
     - vetshadow
     - whitespace
+
+linters-settings:
+  misspell:
+    ignore-words:
+      - unknwon
```

Comment:
unknown?

Reply:
There is a library `"github.com/unknwon/com"` and spellcheck think this is misspell :)